docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

```diff
@@ -16,32 +16,533 @@
    openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Kaldi
    openvino_docs_MO_DG_prepare_model_convert_model_tutorials
 
-.. meta::
-   :description: In OpenVINO, ONNX, PaddlePaddle, TensorFlow and TensorFlow Lite 
-                 models do not require any prior conversion, while MxNet, Caffe and Kaldi do.
 
+**OpenVINO IR (Intermediate Representation)** - the proprietary and default format of OpenVINO, benefiting from the full extent of its features. All other model formats presented below will ultimately be converted to :doc:`OpenVINO IR <openvino_ir>`.
 
-**OpenVINO IR (Intermediate Representation)** - the proprietary format of OpenVINO™, benefiting from the full extent of its features.
+**ONNX, Pytorch, PaddlePaddle, TensorFlow, and TensorFlow Lite** may be used without any prior conversion and can be read directly by OpenVINO Runtime API by the use of ``read_model()`` or ``compile_model()``. Additional adjustment for the model can be performed using the ``convert_model()`` method, which allows you to set shapes, types or the layout of model inputs, cut parts of the model, freeze inputs etc. The detailed information of capabilities of  ``convert_model()`` can be found in :doc:`this <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>` article.
 
-**ONNX, PaddlePaddle, TensorFlow, TensorFlow Lite** - formats supported directly, which means they can be used with 
-OpenVINO Runtime without any prior conversion. For a guide on how to run inference on ONNX, PaddlePaddle, or TensorFlow, 
-see how to :doc:`Integrate OpenVINO™ with Your Application <openvino_docs_OV_UG_Integrate_OV_with_your_application>`.
+Below you will find code examples for each method, for all supported model formats.
 
-**MXNet, Caffe, Kaldi** - legacy formats that need to be converted to OpenVINO IR before running inference. 
-The model conversion in some cases may involve intermediate steps. OpenVINO is currently proceeding 
-**to deprecate these formats** and **remove their support entirely in the future**.
+.. tab-set::
 
+   .. tab-item:: PyTorch
+      :sync: torch
 
-Refer to the following articles for details on conversion for different formats and models:
+      .. tab-set::
+
+         .. tab-item:: Python
+            :sync: py
+
+            * The ``convert_model()`` method:
+
+              When you use the ``convert_model()`` method, you have more control and you can specify additional adjustments for ``ov.Model``. The ``read_model()`` and ``compile_model()`` methods are easier to use, however, they do not have such capabilities. With ``ov.Model`` you can choose to optimize, compile and run inference on it or serialize it into a file for subsequent use.
```

Comment:
We should say here that only `convert_model` method is applicable for conversion `PyTorch` models.

Reply:
I have suggested a change in the https://github.com/openvinotoolkit/openvino/pull/18495/commits/9afe744a3d8ed1ceff5797adac1f3f85ab8ce1e5 commit.